tox.ini

```diff
@@ -18,6 +18,14 @@ deps =
   -r{toxinidir}/docs_guidelines/requirements-extra.txt
 commands = sphinx-build -b html -W {posargs} docs_guidelines/ docs_guidelines/_build/html
 
+[testenv:doctest]
+usedevelop = True
+install_command = pip install -U {opts} {packages}
+deps =
+  -r{toxinidir}/requirements-dev.txt
+  -r docs_guidelines/requirements-extra.txt
+commands = sphinx-build -b doctest {posargs} docs_guidelines/ docs_guidelines/_build/html
```

Comment:
Do we want the `output.txt` in `docs_guidelines/_build/html`? Not sure if it's actually related to the HTML build. I'd put it in a new folder called `docs_guidelines/_build/doctest` or in `docs_guidelines/_build`

Reply:
It contains exactly what's shown in the command line interface, you are right about that. However I'm not sure you can avoid storing it somewhere, as `sphinx-build` forces you to include an `outputdir`, in which the `output.txt` gets inserted. 

With the current configuration you'd obtain a file called `docs_guidelines/_build/html/output.txt` after running the `doctest`.

Reply:
I see. I misread your comment to be about the file rather than the path. I will change the path.